wordpressorg.dev/provision/wp-config.php

```diff
@@ -83,6 +82,13 @@
 	define( 'SITE_ID_CURRENT_SITE', 1 );
 	define( 'BLOG_ID_CURRENT_SITE', 1 );
 	define( 'SUBDOMAIN_INSTALL',    false );
+	
+} elseif ( $_SERVER['HTTP_HOST'] === 'translate.wordpressorg.dev' ) {
+	define( 'GLOTPRESS_TABLE_PREFIX', 'translate_' );
+	define( 'DB_CHARSET', 'latin1' );
+	define( 'GP_TMPL_PATH',  WPORGPATH . 'wp-content/plugins/wporg-gp-customizations/templates/' );
+	define( 'GP_URL_BASE', '' );
+	$gp_table_prefix = GLOTPRESS_TABLE_PREFIX;
```

Comment:
This should be part of the else branch below. You can add it below `define( 'SUBDOMAIN_INSTALL', true );`.

FWIW: w.org uses latin1 for the whole global.wordpress.org network.